doc/usermanual/po/de.po

```diff
@@ -715,7 +716,8 @@ msgstr ""
 "Libraries, die zur Laufzeit geladen werden. Mit dieser Option können Sie "
 "darktable mitteilen, wo es nach diesen Bibliotheken suchen soll. Die "
 "Voreinstellung hängt von Ihrer Installation ab; typische Orte sind <quote>/"
-"opt/darktable/lib/darktable/</quote> und <quote>/usr/lib/darktable/</quote>."
+"opt/darktable/lib/darktable/</quote> und <quote>/usr/lib/darktable/</quote>. "
+"# Hier ist ein anderer Pfad angegeben als im englischen Original."
```

Comment:
Path is changed according to the English version.

Please do not insert comments directly into the translation. Instead mark the translation as not finished and insert a comment via poedit (comment button)